our_site/planner/models.py

```diff
@@ -1,12 +1,13 @@
 from django.db import models
 from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
 
 
 class Profile(models.Model):
-    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    user = models.OneToOneField(User, on_delete=models.CASCADE, related_name="profile")
     roles = models.ManyToManyField("Role", related_name="fulfilled_by")
 
-
```

Comment:
Are we changing our code styling to have only one newline between class definitions? Would it be worth talking about some styling rules at some point? It could be not really important if no one cares, or we could also lay out some common ideas if people care about that.